packages/google-tag-manager/index.js

```diff
@@ -48,10 +49,15 @@ module.exports = async function nuxtTagManager(_options) {
   // Register plugin
   this.addPlugin({
     src: path.resolve(__dirname, 'plugin.js'),
-    fileName: 'google-tag-manager.js',
+    fileName: path.join('google-tag-manager', 'plugin.js'),
```

Comment:
I think the class can be also defined inside the plugin file to prevent creating an extra directory. It is small enough to fit.